cfpb/consumerfinance.gov
# Update DS, axe-core, postcss, cypress, prettier, stylelint, snyk

## Changes

- Update DS, axe-core, postcss, cypress, prettier, stylelint
- Bring in mixin name changes from https://github.com/cfpb/design-system/pull/1993
- Autofix linter issues.


## How to test this PR

1. `yarn install && yarn build` and visit http://localhost:8000/data-research/small-business-lending/filing-instructions-guide/2024-guide-archive-v1/ and see that archive banner visited link is not hidden.

## Screenshot

Before:
<img width="382" alt="Screenshot 2024-07-31 at 9 53 32 AM" src="https://github.com/user-attachments/assets/10b62aab-f66c-42a3-ba45-096875f9572d">

After:
<img width="409" alt="Screenshot 2024-07-31 at 9 53 22 AM" src="https://github.com/user-attachments/assets/73757203-9e2a-490d-bb66-44e2ec931254">
